__tests__/boot.ts

```diff
@@ -399,6 +407,105 @@ describe('logged in boot', () => {
   });
 });
 
+describe('boot marketing cta', () => {
```

Comment:
Do you maybe want to add one with SLEEPING from start?

Reply:
Added one